src/app/common/hooks/use-onclickoutside.ts

```diff
@@ -41,6 +45,12 @@ export function useOnClickOutside(ref: RefObject<HTMLElement>, handler: Handler
         return;
       }
 
+      // If the click is inside an exception, don't call the handler
+      const exceptionsElements = exceptionIds?.map(el => document.getElementById(el));
```

Comment:
```suggestion
      const exceptionsElements = exceptionIds.map(el => document.getElementById(el));
```
Don't need the `?` as you're setting a default array param right?

Wonder if, to avoid the comment, maybe you could make a fn so the code reads the same?

`if (clickedElementIsAnException())` ?

Reply:
yeah, makes sense. fixed